tests/storage/cdi_upload/test_upload_virtctl.py

```
@@ -53,6 +55,30 @@ def skip_no_reencrypt_route(upload_proxy_route):
 
+@pytest.fixture(scope="function")
+def cdi_uploadproxy_url(admin_client, hco_namespace):
+    """
+    Get CDI upload proxy URL.
+
+    On IPv6 single-stack clusters, set up port-forward to the CDI upload proxy service
+    and return localhost URL instead of the route (which resolves to IPv4).
+    """
+    if is_ipv6_single_stack_cluster():
+        port = 8443
+        port_forward_process = subprocess.Popen(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)